app/controllers/propertyLinking/ChooseEvidenceController.scala

```diff
@@ -51,18 +56,41 @@ class ChooseEvidenceController @Inject()(
 
   def show: Action[AnyContent] = authenticatedAction.andThen(withLinkingSession).async { implicit request =>
     logger.debug("show choose evidence page")
-    businessRatesAttachmentService
-      .persistSessionData(request.ses, UploadEvidenceData.empty)
-      .map(_ => Ok(chooseEvidenceView(ChooseEvidence.form, request.ses.clientDetails)))
+    for {
+      startDate <- propertyLinkingService.findEarliestStartDate(request.ses.uarn)
```

Comment:
where are you using this "startDate" ?

Reply:
We are using to make decision of backlink redirecting correct page